### DISCUSSION TITLE
Получить данные $#nu пользователя через API

### DISCUSSION THREAD

**Discussion by DaveMaks** [totumonline/totum-ru-issues-and-discussions#319] [Как это сделать?] [answered]
Подскажите пожалуйста, 
как правильно получить данные о текущем (указанный в "auth": {} ) пользователе (name/email/roles) через API, я сейчас вижу выход создать служебную таблицу с пре-фильтром $#nu. Правильный ли ход мысли, или есть какойто более простое решение?

**Comment by ttm-alexeypolunin** [comment, marked as answer]
Просто вызвать $#nu в коде ремоута и у вас будет id пользователя Remote или API:

Микро-пример:
<img width="857" alt="Снимок экрана 2023-01-25 в 01 27 10" src="https://user-images.githubusercontent.com/55755565/214435235-6a49fd7f-cdfa-4927-bb8a-26b9b1fab6aa.png">


Потом сделать selectRow из таблицы users по этому id.
